packages/utils/src/modern.js

```diff
@@ -21,20 +33,16 @@ const isModernBrowser = (ua) => {
   return Boolean(modernBrowsers[browser.name] && semver.gte(browserVersion, modernBrowsers[browser.name]))
 }
 
-const detectModernBrowser = ({ socket = {}, headers }) => {
-  if (socket.isModernBrowser === undefined) {
-    const ua = headers && headers['user-agent']
-    socket.isModernBrowser = isModernBrowser(ua)
+export const isModernRequest = (req, options) => {
```

Comment:
As this is a shared util, isn't better idea to not passing entire options object?